pennylane_lightning/src/simulator/SimulatorUtil.cpp

```diff
@@ -0,0 +1,76 @@
+#include "SimulatorUtil.hpp"
+#include "AvailableKernels.hpp"
+
+#include "Util.hpp"
+
+namespace Pennylane::IndicesUtil {
```

Comment:
Should this be also renamed as 
```suggestion
namespace Pennylane::SimulatorUtil {
```
?

Reply:
As I removed this file into `gates` subdirectory, it is not `GateUtil`.